func/func_server.h

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include <grpc/grpc.h>
+using grpc::Server;
+using grpc::ServerBuilder;
+using grpc::ServerContext;
+
+class FuncServiceImpl final : public FuncService::Service {
+public:
```

Comment:
public and private should be indented 1 space

Reply:
ok

Reply:
It appears these haven't changed?

Reply:
It seems I forgot to push these changes. Sorry!